docs/pages/billing/faq.mdx

```diff
@@ -112,4 +112,8 @@ To view the cost of a large build, go to [Billing](https://expo.dev/settings/bil
 
 ### How do I increase build concurrencies on my account?
 
-[Contact us](https://expo.dev/contact) for more information.
+If you're already subscribed to a paid EAS plan, you can buy additional concurrencies in [the Add-ons section on the Billing](https://expo.dev/settings/billing#addons) page.
```

Comment:
```suggestion
If you're already subscribed to a paid EAS plan, you can buy additional concurrencies in the [**Add-ons** section](https://expo.dev/settings/billing#addons) under **Billing**.
```

Let's make settings/pages name bold as this is the pattern we follow in this doc for describing paths to other settings/areas of billing page.